### DISCUSSION TITLE
Changing bed size in Releases?

### DISCUSSION THREAD

**Discussion by smg492** [Jyers/Marlin#929] [Q&A] [answered]
The BL touch version of the newest release is working great, however I just upgraded my frame and bed size.  Is there an easy way I can keep this firmware and change the bed size?

**Comment by smooreace** [comment]
Compile the firmware for your bed

**Comment by Jyers** [comment, marked as answer]
Unfortunately the only way is to compile your own firmware, if you want to make your life a bit easier you can become a patreon and use the auto-compile service, but either way you'll need to know your way around a config file.
